src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs

```diff
@@ -105,17 +105,17 @@ void OnTextPropertyChanged()
 			}
 			tokenSource = new CancellationTokenSource();
 
-			_ = Task.Delay(StoppedTypingTimeThreshold, tokenSource.Token)
+			Task.Delay(StoppedTypingTimeThreshold, tokenSource.Token)
```

Comment:
Regarding not awaited tasks, Here's another one

Reply:
I don't understand why do we need await it here and there

https://www.tabsoverspaces.com/233659-do-not-await-what-does-not-need-to-be-awaited

Reply:
This one is leveraging `ContinueWith`, so I'll add the following code to rethrow the  `Exception` if the `Task` faulted:

```
if (task.IsFaulted)
    throw task.Exception;
```

Reply:
Should we check if the task was cancelled as well?

Reply:
That check is already there: `task.Status == TaskStatus.Canceled`. I think that's why `ContinueWith` was used in the first place

Reply:
I didn't see that hehe

Reply:
> I don't understand why do we need await it here and there
> 
> https://www.tabsoverspaces.com/233659-do-not-await-what-does-not-need-to-be-awaited

@AndreiMisiukevich We don't return this task from the method, so it's never awaited. This article talks about the case where we return the task, so it will be awaited, but later:

![image](https://user-images.githubusercontent.com/6609929/110224652-0c5e0c00-7ee6-11eb-8742-22cc2bddea0c.png)

Reply:
So you are going to crash the app if the task is canceled?

Reply:
> ![image](https://user-images.githubusercontent.com/6609929/110224652-0c5e0c00-7ee6-11eb-8742-22cc2bddea0c.png)

Yes, it's never awaited because it's an event handler.